src/services/ticketing-service/adapters/odoo-adapter.ts

```diff
@@ -0,0 +1,84 @@
+import Odoo from 'odoo-await'
+import Config from '../../../common/config'
+import { transformKeysToCamelCase } from '../../../utils/transformStrings'
+
+interface OdooGetTicket {
```

Comment:
What do you think about having one interface called `TicketOdoo` which is for both fetching and writing to Odoo and the one called just Ticket for the camelCase one? 
Then we could extend it if it's something specific in the get-ing and post-ing? might be a really bad idea, and maybe this makes more sense to look into in the 1335-story? Or maybe we could have a `TicketIn` and a `TicketOut`? Let's discuss :)